panther_ioc_rules/sunburst_fqdn_iocs.py

```diff
@@ -0,0 +1,9 @@
+from panther_iocs import sunburst_fqdn_ioc_match
+
+
+def rule(event):
+    return sunburst_fqdn_ioc_match(event)
+
+
+def title(event):
+    return "Sunburst Indicator of Compromise (FQDN) Detected"
```

Comment:
Let's gussy this up a bit by adding at least the `p_any_domains` field to this. Ideally, we could specify the exact FQDN that matched. You could take the intersection off the `p_any_domains` field and the ioc's field to see which domain it was exactly (maybe with some escaping e.g. replace `.`  with `[.]` so that someone doesn't actually click a known malicious domain).

Same comment for the other rules.